content/en/examples/pods/inject/dapi-volume-resources.yaml

```diff
@@ -30,7 +30,6 @@ spec:
       volumeMounts:
         - name: podinfo
           mountPath: /etc/podinfo
-          readOnly: false
```

Comment:
Hello @ImJasonH . Does the `readOnly` field default to _false or unspecified_ in VolumeMounts? If so, then removing this setting should be okay.
nit: Is your commit message correct, `even if readOnly:true is set` vs `even if readOnly:false`?

Reply:
The field is a `bool`, so it defaults to `false` (not `unspecified`). Amended the commit message, which was indeed incorrect.

Thanks for catching that!

Reply:
OK, thanks for clarifying. Probably should add a note to the task.
/lgtm